app/views/docs/command-line-deployment.phtml

```diff
@@ -110,7 +122,7 @@ The Apprite CLI allows you to create and deploy collections and functions to you
         <tr>
             <td><code>runtime</td>
             <td>string</td>
-            <td>Function runtime, must be enabled. <a href="docs/functions#supportedRuntimes">Learn more about runtimes</a>.</td>
+            <td>Function runtime, must be enabled. <a href="/docs/functions#supportedRuntimes">Learn more about runtimes</a>.</td>
```

Comment:
I added `/` here for consistency cause we do it everywhere else. Functionally identical, though

Reply:
👍